lib/pastes.js

```diff
@@ -45,6 +45,22 @@ async function getRandomPaste() {
   return paste;
 }
 
+async function incrementPastaPoints(pasteId) {
+  const pastesCollection = await getCollection(collectionName);
+  const objectId = new ObjectID.createFromHexString(pasteId);
+  const paste = await getPaste(pasteId);
+  if (paste.pastaPoints) {
+    paste.pastaPoints++;
+  } else {
+    paste.pastaPoints = 1;
+  }
+  pastesCollection.update(
+    { _id: objectId },
+    { $set: { pastaPoints: paste.pastaPoints } }
```

Comment:
An alternative is the `$inc` operator:
https://docs.mongodb.com/manual/reference/operator/update/inc/index.html